src/kf_scripts/cull_restarts.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import datetime
+import os
+
+
+def cull_restarts(jacobian_runs_dir, posterior_dir, start_day, end_day):
+    """
+    GEOS-Chem bugs mean we need to output daily restart files instead of just one
+    at the end of each week. This function culls the extra restarts we don't need.
+
```

Comment:
@msulprizio Is this right? Can't we set the restart output frequency to End?

Reply:
This might be outdated by 2+ years (or maybe it was always wrong).